### ISSUE TITLE
CMAKE: fix pipeline builds for windows

### ISSUE THREAD

**Issue by mgerhardy** [PadWorld-Entertainment/worldofpadman#269]
They are not properly linking the msvc runtime libs.

**Comment by kai-li-wop** [comment]
Finally I had the chance to test the latest build on Windows 10, but running on some very old Intel Pentium 4 hardware with some integrated graphics with 64MB VRAM. Apart from heavy loading times and 5fps in the main menu, I was able to run the game with OpenGL1 without needing the Microsoft Redistributable. So this fixes the build pipeline in my case. I will wait for feedback from ENTE before closing this issue.

Apart from that, I ran into the problem that OpenGL2 was not supported and I had to manually switch back to OpenGL1 by adding +set cl_renderer opengl1 as a startup command line parameter. It seems that switching back to save video parameters via the save video parameters dialog after restarting WoP does not change the renderer to openGL1 if OpenGL2 is not supported. Does ioquake3 has a workaround for this @zturtleman?

![Image](https://github.com/user-attachments/assets/e5958b17-5565-4f7b-9f25-e8d2d2a1f003)

![Image](https://github.com/user-attachments/assets/13cdc5f1-2b1a-42d7-908b-e9639715aa31)
